### ISSUE TITLE
Unable to start container

### ISSUE THREAD

**Issue by jld3103** [ubports/crossbuilder#20]
```
jld3103@pippin:~/Desktop/anbox$ crossbuilder dependencies
Creating LXD container anbox-usdk-16-04-amd64-armhf-dev using ubports-sdk:ubuntu-sdk-16.04-amd64-armhf-dev.
Creating anbox-usdk-16-04-amd64-armhf-dev
Error: Failed to run: /snap/lxd/current/bin/lxd forkstart anbox-usdk-16-04-amd64-armhf-dev /var/snap/lxd/common/lxd/containers /var/snap/lxd/common/lxd/logs/anbox-usdk-16-04-amd64-armhf-dev/lxc.conf: 
Try `lxc info --show-log anbox-usdk-16-04-amd64-armhf-dev` for more info
jld3103@pippin:~/Desktop/anbox$ lxc info --show-log anbox-usdk-16-04-amd64-armhf-dev
Name: anbox-usdk-16-04-amd64-armhf-dev
Remote: unix://
Architecture: x86_64
Created: 2019/07/07 06:48 UTC
Status: Stopped
Type: persistent
Profiles: default

Log:

lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.343 ERROR    start - start.c:proc_pidfd_open:1607 - Function not implemented - Failed to send signal through pidfd
lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.344 ERROR    conf - conf.c:write_id_mapping:2763 - Invalid argument - Failed to write uid mapping to "/proc/8412/uid_map"
lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.344 ERROR    conf - conf.c:lxc_map_ids:2962 - Failed to write mapping: 0 1000000 1000000000
150000 1000 1

lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.344 ERROR    start - start.c:lxc_spawn:1770 - Failed to set up id mapping.
lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.387 WARN     network - network.c:lxc_delete_network_priv:2743 - Failed to remove interface "veth046PJF" from "lxdbr0"
lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.387 ERROR    lxccontainer - lxccontainer.c:wait_on_daemonized_start:851 - Received container state "ABORTING" instead of "RUNNING"
lxc anbox-usdk-16-04-amd64-armhf-dev 20190707064911.387 ERROR    start - start.c:__lxc_start:2019 - Failed to spawn container "anbox-usdk-16-04-amd64-armhf-dev"
lxc 20190707064911.413 WARN     commands - commands.c:lxc_cmd_rsp_recv:135 - Connection reset by peer - Failed to receive response for command "get_state"
```

**Comment by jld3103** [comment]
When using ```--privileged``` from #17 it works.

**Comment by UniversalSuperBox** [comment]
Fixed by #17
